packages/pug/lib/index.js

```diff
@@ -172,7 +172,7 @@ function compileBody(str, options){
 
   // Compile
   ast = applyPlugins(ast, options, plugins, 'preCodeGen');
-  var js = generateCode(ast, {
+  var js = findReplacementFunc(plugins, 'codeGen', generateCode)(ast, {
```

Comment:
If you made this:

```suggestion
  var js = (findReplacementFunc(plugins, 'codeGen') || generateCode)(ast, {
```

You wouldn't need any changes to `findReplacementFunc` at all.

Reply:
I made the suggested change and removed the changes to `findReplacementFunc()`.